src/TabBar.js

```diff
@@ -17,7 +18,7 @@ const styles = StyleSheet.create({
     flex: 1,
   },
   scroll: {
-    overflow: 'scroll',
+    overflow: Platform.OS === 'web' ? 'none' : 'scroll',
```

Comment:
`none` is not a valid value for overflow on web, maybe you meant `auto`?

Valid values are `visible`, `hidden`, `scroll` and `auto`.